benchmark/_dummy-benchmarker.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
I don't like `noop` because `noop` means "no operator". A `noop` should do nothing whatsoever. But that's not the case here.

In this case, this is a test double. to `_test-double-benchmarker` maybe? Sometimes people call certain types of test doubles `mocks` and other types `stubs`, so `mock-benchmarker` or `stub-benchmarker` might also be appropriate. (I stick to `test double` so that I don't have to worry about whether or not something is a mock or a stub.)